radix-transactions/src/builder/transaction_builder.rs

```diff
@@ -457,7 +517,7 @@ impl TransactionV2Builder {
 
     pub fn create_signed_transaction_intent(&mut self) -> &SignedTransactionIntentV2 {
         if self.signed_intent.is_none() {
-            self.create_intent_and_subintent_signatures();
+            self.create_intent_and_subintent_info();
             let (root_intent, subintent_signatures) =
                 mem::take(&mut self.intent_and_subintent_signatures).unwrap();
```

Comment:
1. `Option::take()` instead of `mem::take()` here and below?

2. How about using `expect()` similarly to other places?
eg.
```
expect("Root intent and subintent signatures must be provided before this action is performed");
```

Reply:
Ahh this is because it's guaranteed to exist from the line above -- rather than being a user-facing message. But I'll add an expect for it anyway :)